modules/api-support-tiers.adoc

```diff
@@ -0,0 +1,130 @@
+// Module included in the following assemblies:
+//
+// * rest_api/understanding-api-support-tiers.adoc
+
+[id="api-tiers_{context}"]
+= API tiers
+
+All commercially supported APIs, components, and features are associated under one of the following support levels:
+
+[discrete]
+[id="api-tier-1_{context}"]
+== API tier 1
+APIs and application operating environments (AOEs) are stable within a major release for a minimum of 12 months or 3 minor releases from the announcement of deprecation, whichever is longer. The release that introduces a new or revised API or AOE, and the two following minor releases (n, n+1, n+2).
+
+[discrete]
+[id="api-tier-2_{context}"]
+== API tier 2
+APIs and AOEs are stable within a major release for a minimum of 9 months or 3 minor releases from the announcement of deprecation, whichever is longer. API tier 2 is the default for components in {product-title} where not identified.
+
+[discrete]
+[id="api-tier-3_{context}"]
+== API tier 3
+This level applies to languages, tools, applications, and optional Operators included with {product-title} through Operator Hub. Each component will specify a lifetime during which the API and AOE will be supported. Newer versions of language runtime specific components will attempt to be as API and AOE compatible from version to version as possible. Version to version compatibility is not guaranteed, however.
```

Comment:
when we say "Version to version" here, we're referring to minor versions, right?  There is still an overall claim elsewhere that we don't break stuff in z-releases which i assume still applies here.

Reply:
derek says yes we can explicitly say minor versions